modules/ossm-kiali-service-mesh.adoc

```diff
@@ -14,4 +13,4 @@ Installing Kiali  via the Service Mesh on {product-title} differs from community
 * Ingress has been enabled by default.
 * Updates have been made to the Kiali ConfigMap.
 * Updates have been made to the ClusterRole settings for Kiali.
-* Do not edit the ConfigMap or the Kiali custom resource files as those changes might be overwritten by the {SMProductShortName} or Kiali Operators. All configuration for Kiali running on {SMProductName} is done in the `ServiceMeshControlPlane` custom resource file and there are limited configuration options. Updating the Operator files should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the operator files should be restricted to those users with `dedicated-admin` privileges.
+* Do not edit the ConfigMap, as your changes might be overwritten by the {SMProductShortName} or Kiali Operators. Files that are managed by the Kiali Operator have a `kiali.io/` label or annotation. Updating the Operator files should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the Operator files should be restricted to those users with `dedicated-admin` privileges.
```

Comment:
Do not edit the ConfigMap, as your changes might be overwritten by the {SMProductShortName} or Kiali Operators. Files that the Kiali Operator manages have a `kiali.io/` label or annotation. Updating the Operator files should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the Operator files should be restricted to those users with `dedicated-admin` privileges.